ginlsat/ginlsat.go

```diff
@@ -125,15 +124,13 @@ func (lsatmiddleware *GinLsatMiddleware) SetLSATHeader(c *gin.Context) {
 	}
 	invoice, paymentHash, err := LNClientConn.GenerateInvoice(ctx, lnInvoice, c.Request)
 	if err != nil {
-		c.Error(err)
 		c.Set("LSAT", &LsatInfo{
 			Error: err,
 		})
 		return
 	}
 	macaroonString, err := macaroonutils.GetMacaroonAsString(paymentHash)
 	if err != nil {
-		c.Error(err)
 		c.Set("LSAT", &LsatInfo{
```

Comment:
Shouldn't we add a type: "ERROR" here or something?

Reply:
We have already put the error into LSAT info that can be served along with the request response (if any error occurred). I found it a bit irrelevant when the error got logged with the test results for the invalid macaroon test. I can undo the [commit ](https://github.com/getAlby/gin-lsat/pull/17/commits/e6e5a60b3d22da757c757ef4d34bcc7538dc509d) or even we can implement a logging mechanism that can put all the generated errors with the timestamp in a separate file.